### ISSUE TITLE
Display info about upcoming booking

### ISSUE THREAD

**Issue by jaroslav-kubicek** [kiwicom/smart-faq#28]
**Acceptance criteria:**
- a user can use the input to search in static FAQs
- no breadcrumb navigation, no categories, just list of matching FAQ is displayed

<img width="938" alt="screen shot 2018-03-21 at 11 17 23" src="https://user-images.githubusercontent.com/2174629/37707032-7c6ddae4-2cf9-11e8-8f47-2db30ad1d29c.png">

https://marvelapp.com/7iihb90/screen/39265596

**Comment by MagdalenaAlmanova** [comment]
When I logged in, it showed me this:

![screen shot 2018-04-19 at 21 26 49](https://user-images.githubusercontent.com/37298895/39013853-f44ab560-4418-11e8-9469-1b35bb04f236.png)

I do not know if this is not because I have no booking. I'm probably going to need a test account.

Meggy

**Comment by MagdalenaAlmanova** [comment]
I found some mistakes. Such as, the layout of the screen was not completed, the title "**Help**" is written on the main page, not in FAQ. You can see this in the screenshot. 
![screen shot 2018-04-19 at 21 26 49](https://user-images.githubusercontent.com/37298895/39269387-f4ffe32c-48d2-11e8-8da6-88d7bec151f8.png)

Secondly, sign out, select other bookings, outbound and return were not in bold font type like in zeplin.

Next, there was **a plane** between two destinations. I think, that it was not in zeplin. There is just **an** **arrow**. 
**Demo v.**
![screen shot 2018-04-25 at 17 57 21](https://user-images.githubusercontent.com/37298895/39269707-d8ac8e86-48d3-11e8-9f4d-8f17c27bd967.png)

**_Prototype v._**
![screen shot 2018-04-25 at 17 57 30](https://user-images.githubusercontent.com/37298895/39269740-f7438930-48d3-11e8-8c6c-00e56ef7d4b7.png)

The last one, the width of the box where the outbound or return is shown, was not correct and there was a line at the bottom which should not be there. 

It was tested in **_https://introduce-smart-faq.fe.staging.kiwi.com/en/_** 

Thank you,
Meggy

**Comment by git-toni** [comment]
Thanks Magda :) You are right about this points at https://introduce-smart-faq.fe.staging.kiwi.com/en/ . 

However I think the latest version is at https://kiwicom.github.io/smart-faq/ and it should be tested there. Take into account that in that URL we've got a problem with the spinning plane loader ( it is not spinning :sweat_smile: ). 

Also regarding the `arrow` vs `plane` icon between destinations, we are still waiting for a new release of the icons library, so cannot add the arrow yet.

As for the outbound/return box, can you test that this is correct in that URL pls?

**Comment by MagdalenaAlmanova** [comment]
Hmm,  of course, I can. But @jalbertsr wrote me that I can test at https://introduce-smart-faq.fe.staging.kiwi.com/en/ but no problem. I will test at https://kiwicom.github.io/smart-faq/  😄

**Comment by git-toni** [comment]
You are right, he just told me as well, my bad. :) I think https://kiwicom.github.io/smart-faq/ will be more up-to-date in general. I guess we can talk about this on the next meeting.

**Comment by MagdalenaAlmanova** [comment]
Ok, so I was testing at https://kiwicom.github.io/smart-faq/ and I discovered some small mistakes. I read that icons will be new.  The different icon was in "blue" textbox, see on the screenshot. 
**_Demo v._**
![screen shot 2018-04-26 at 19 17 55](https://user-images.githubusercontent.com/37298895/39322978-aec5a0ea-498b-11e8-9e05-b5ceeb8649cb.png)
**_Prototype v._**
![screen shot 2018-04-26 at 19 18 03](https://user-images.githubusercontent.com/37298895/39323005-bf23da24-498b-11e8-9e2b-f572d02c837b.png)

and icon of exit **"x"** was bigger than in the prototype v.

Appart from icons, there were other mistakes. Like **"FULL FAQ site"** and **"Sing out"** was not in line.
 **_Demo v._**
![screen shot 2018-04-26 at 19 19 47](https://user-images.githubusercontent.com/37298895/39323180-26cf6fd0-498c-11e8-9687-e315d7213fab.png)
 
The next one, **"Upcoming booking"** and **"What can we help you with"** was not in line. I think, that **"upcoming booking"** should be a bit higher. 
**_Demo v._**
![screen shot 2018-04-26 at 20 04 41](https://user-images.githubusercontent.com/37298895/39323656-7fbf78dc-498d-11e8-98e1-90a61f574dc2.png)

 **_Prototype v._**
![screen shot 2018-04-26 at 20 04 52](https://user-images.githubusercontent.com/37298895/39323689-8ff82a50-498d-11e8-9e00-ff29b63d6e25.png)

Only in **Opera**, "Full FAQ site" and the icon was not in line.
**_Demo v._**
![screen shot 2018-04-26 at 19 29 33](https://user-images.githubusercontent.com/37298895/39323780-ea98073c-498d-11e8-8476-f55e32e44da1.png)

The last one, again I saw this in another issue. The icon "magnifier" was not in the middle of the box.  
**_Demo v._**
![screen shot 2018-04-26 at 19 19 34](https://user-images.githubusercontent.com/37298895/39324124-fc2d463c-498e-11e8-9772-73f418877a6a.png)

**_Prototype v._**
![screen shot 2018-04-26 at 19 19 25](https://user-images.githubusercontent.com/37298895/39324150-18bc1436-498f-11e8-9898-5f6bf95cdc7e.png)

I do not know If that was part of this issue, but I write this: some subcategories of FAQ were out of the box.

Thanks, 
Meggy 😄

**Comment by git-toni** [comment]
Thanks @MagdalenaAlmanova, 
A few doubts just to make sure what bugs have been fixed since you saw them and which need to be addressed: 

- Regarding the *Upcoming booking* and What can we help you with alignment. What design/prototype are you basing it with? I don't see them aligned in [this design](https://app.zeplin.io/project/5a8e8992554d385c76ad5aeb/screen/5aaac0e6c41e3efb0dcd168a)
![image](https://user-images.githubusercontent.com/2466887/39930752-d7876674-553b-11e8-85a9-0a482d115067.png)
- The *Full FAQ site" + "Sign out" mis-alignment. I see it fine in my browser 
![image](https://user-images.githubusercontent.com/2466887/39930874-254361f6-553c-11e8-8af5-5815a6d9cf14.png)

- Will fix the Magnifier mis-alignment tho, that is still present!

**Comment by MagdalenaAlmanova** [comment]
This will be edited. 

Thank you for your answer, about the first question: It was an old "prototype v.". I think that it is Ok now. :) 

But I cannot sing in, it showed me the white page. I am going to try it tomorrow. 

M.

**Comment by git-toni** [comment]
Yep I think the URL https://introduce-smart-faq.fe.staging.kiwi.com/ is not updated yet, it's also giving me problems.

**Comment by MagdalenaAlmanova** [comment]
ok
